UI/quest_counter.csv

```diff
@@ -669,72 +669,72 @@ QuestTypeName_EP5_Basnet#0,"""バスタークエスト【ストーリー】"""
 QuestTypeName_Omnibus_Event#0,"""イベントクエスト"""
 QuestTypeName_Omnibus_Battle#0,"""Misiónes de Historia"""
 Omnibus_FilterChapter#0,"""$(0).$(1)"""
-Omnibus_EP1_Name_All#0,"""すべて"""
+Omnibus_EP1_Name_All#0,"""Todo"""
 Omnibus_EP1_Name_NotClear#0,"""Misión no completada"""
-Omnibus_EP1_Name_00#0,"""1章：マターボードの導き"""
-Omnibus_EP1_Name_01#0,"""２章：龍族との邂逅"""
-Omnibus_EP1_Name_02#0,"""３章：小さな恩人"""
-Omnibus_EP1_Name_03#0,"""４章：壊れた創世器"""
-Omnibus_EP1_Name_04#0,"""５章：二つ目の欠片"""
-Omnibus_EP1_Name_05#0,"""６章：最後の欠片"""
-Omnibus_EP1_Name_06#0,"""７章：緊急事態"""
-Omnibus_EP1_Name_07#0,"""８章：【巨躯】復活"""
-Omnibus_EP1_Name_08#0,"""９章：アークスとダークファルス"""
-Omnibus_EP1_Name_09#0,"""１０章：アイドルと暴走龍"""
-Omnibus_EP2_Name_All#0,"""すべて"""
+Omnibus_EP1_Name_00#0,"""Cap.1：Orientación de Mother Board"""
+Omnibus_EP1_Name_01#0,"""Cap.2：Encuentro de dragones"""
+Omnibus_EP1_Name_02#0,"""Cap.3：Pequeño Benefactor"""
+Omnibus_EP1_Name_03#0,"""Cap.4：La Génesis quebrada"""
+Omnibus_EP1_Name_04#0,"""Cap.5：El segundo fragmento"""
+Omnibus_EP1_Name_05#0,"""Cap.6：El fragmento final"""
+Omnibus_EP1_Name_06#0,"""Cap.7：Estado de emergencia"""
+Omnibus_EP1_Name_07#0,"""Cap.8：La resurreccion de [Elder]"""
+Omnibus_EP1_Name_08#0,"""Cap.9：ARKS y Dark Falz"""
+Omnibus_EP1_Name_09#0,"""Cap.10：La ídol y el dragón fugitivo"""
+Omnibus_EP2_Name_All#0,"""Todo"""
 Omnibus_EP2_Name_NotClear#0,"""Misión no completada"""
-Omnibus_EP2_Name_01#0,"""1章：遺された者達"""
-Omnibus_EP2_Name_02#0,"""２章：歴史を変えるために"""
-Omnibus_EP2_Name_03#0,"""３章：閉ざされた真実の先"""
-Omnibus_EP2_Name_04#0,"""４章：再誕の日"""
-Omnibus_EP2_Name_05#0,"""５章：わたしがうまれたそのりゆう"""
-Omnibus_EP2_Name_06#0,"""６章：それぞれの未来"""
-EP5_Name_All#0,"""すべて"""
+Omnibus_EP2_Name_01#0,"""Cap.1：Los que quedaron atrás"""
+Omnibus_EP2_Name_02#0,"""Cap.2：Para cambiar la historial"""
+Omnibus_EP2_Name_03#0,"""Cap.3：El final de la verdad oculta"""
+Omnibus_EP2_Name_04#0,"""Cap.4：Día del Renacimiento"""
+Omnibus_EP2_Name_05#0,"""Cap.5：El motivo de mi nacimiento"""
+Omnibus_EP2_Name_06#0,"""Cap.6：Cada futuro"""
+EP5_Name_All#0,"""Todo"""
 EP5_Name_NotClear#0,"""Misión no completada"""
-EP5_Name_00#0,"""序章：救い伸ばした手の先に"""
-EP5_Name_01#0,"""1章：救世の存在"""
-EP5_Name_02#0,"""２章：滅びゆく国"""
-EP5_Name_03#0,"""３章："""
-EP5_Name_04#0,"""４章："""
-EP5_Name_05#0,"""５章："""
-EP5_Name_06#0,"""６章："""
-EP5_Name_07#0,"""７章："""
-EP5_Name_08#0,"""８章："""
-EP5_Name_09#0,"""９章："""
-EP5_Name_10#0,"""１０章："""
-EP5_Name_11#0,"""１１章："""
-EP5_Name_12#0,"""１２章："""
-EP5_Name_13#0,"""１３章："""
-EP5_Name_14#0,"""１４章："""
-EP5_Name_15#0,"""１５章："""
-EP5_Name_16#0,"""１６章："""
-EP5_Name_17#0,"""１７章："""
-EP5_Name_18#0,"""１８章："""
-EP5_Name_19#0,"""１９章："""
-EP5_Name_20#0,"""２０章："""
-EP5_Name_21#0,"""エピソード：ストラトス"""
-EP5_Name_22#0,"""エピソード：アルマ"""
-EP5_Name_23#0,"""エピソード：シエラ"""
+EP5_Name_00#0,"""Prólogo: Salvación de Vanguardia"""
+EP5_Name_01#0,"""Cap.1：La existencia de la salvación"""
+EP5_Name_02#0,"""Cap.2：La nación maldita"""
+EP5_Name_03#0,"""Cap.3："""
```

Comment:
Please, don't translate EP5_Name_03-20 until been used on the game
Por favor, no traducir EP5_Name_03-20 hasta que sean usados en el juego